src/adaptors/termmax/index.js

```diff
@@ -57,297 +97,479 @@ async function getMerklOpportunities() {
+        chainId,
+        poolAddress: pools.output[i].output,
+        vaultAddress: address,
       });
+    }
+    const tvlUsd = new BigNumber(totalAssetses.output[i].output)
+      .div(new BigNumber(10).pow(decimalses.output[i].output))
+      .times(priceMap.get(assetAddress) || 0)
+      .toNumber();
 
-      const assetAddresses = new Set(asset.output.map((o) => o.output));
-      const priceMap = new Map();
-      {
-        const promises = [];
-        for (const assetAddress of assetAddresses) {
-          const url = new URL(
-            `https://coins.llama.fi/prices/current/${vaultData.chain}:${assetAddress}`
-          );
-          promises.push(
-            axios.get(url).then((response) => {
-              const priceKey = `${vaultData.chain}:${assetAddress}`;
-              priceMap.set(
-                assetAddress,
-                response.data.coins[priceKey]?.price || 0
-              );
-            })
-          );
-        }
-        await Promise.all(promises);
-      }
+    const url = new URL(
+      `https://app.termmax.ts.finance/earn/${alias}/${address.toLowerCase()}`
+    );
+    url.searchParams.set('chain', alias);
 
-      for (let i = 0; i < vaultData.addresses.length; i++) {
-        const address = vaultData.addresses[i];
-        const assetAddress = asset.output[i].output;
-
-        const readableApy = new BigNumber(apy.output[i].output)
-          .div(new BigNumber(10).pow(6))
-          .toNumber();
-        const tvlUsd = new BigNumber(totalAssets.output[i].output)
-          .div(new BigNumber(10).pow(decimals.output[i].output))
-          .times(priceMap.get(assetAddress) || 0)
-          .toNumber();
-
-        const url = new URL(
-          `https://app.termmax.ts.finance/earn/${
-            vaultData.alias
-          }/${address.toLowerCase()}`
-        );
-        url.searchParams.set('chain', vaultData.alias);
-
-        const pool = {
-          pool: `${address}-${chain.toLowerCase()}`,
-          chain,
-          project: 'termmax',
-          symbol: assetNames.output[i].output,
-          tvlUsd,
-          apyBase: readableApy,
-          url: String(url),
-          underlyingTokens: [assetAddress],
-          poolMeta: names.output[i].output,
-        };
-
-        const opportunity = opportunities.find(
-          (o) =>
-            o.chainId === vaultData.chainId &&
-            o.identifier.toLowerCase() === address.toLowerCase()
-        );
-        if (opportunity) {
-          pool.apyReward = opportunity.apr;
-
-          const breakdowns =
-            (opportunity.rewardsRecord &&
-              opportunity.rewardsRecord.breakdowns) ||
-            [];
-          pool.rewardTokens = breakdowns
-            .map((b) => b.token.address)
-            .filter((a) => a);
-        }
-
-        pools.push(pool);
-      }
+    const vault = {
+      pool: `${address}-${chain.toLowerCase()}`,
+      chain,
+      project: 'termmax',
+      symbol: assetNames.output[i].output,
```

Comment:
we should keep the symbol, eg rn the adapter outputs: ` symbol: 'Wrapped liquid staked Ether 2.0',` should just be `WSTETH`